sass/directives/_visual-guides.scss

```diff
@@ -137,8 +130,9 @@ $visual-guide-breakpoint-small: 750px;
     width: $visual-guide-step-size;
     color: $text-color-dark;
 
-    span:after {
+    div:after {
```

Comment:
Is there a class we could specify for the `after` instead, rather than applying these styles to any div? I understand we're assuming the structure of this element, but I'd like to avoid that if at all possible